lib/graphql/float_type.rb

```diff
@@ -1,5 +1,7 @@
 GraphQL::FLOAT_TYPE = GraphQL::ScalarType.define do
   name "Float"
+  description "The `Float` scalar type represents signed double-precision fractional values as specified by [IEEE 754](http://en.wikipedia.org/wiki/IEEE_floating_point)."
```

Comment:
Do these need to start with `"The {x} scalar type represents..."` Could this one just be 

```ruby 

```ruby
description "signed, double-precision fractional value, as specified by [IEEE 754](http://en.wikipedia.org/wiki/IEEE_floating_point)"
```

?